LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
 Copyright (c) 2022 Brian Mayo
+Copyright (c) 2023 Lorenzo Bellina
```

Comment:
Thank you for conserving the copyright. Your changes look cool, is your plan to support variables?
I'm interested to see how your fork evolves. Cool stuff!
@TheRealLorenz

Reply:
Thank you! The project is still a big WIP and I would like to stabilise it before getting to variables, but I'm going to implement them too.